src/native/minipal/cpufeatures.c

```diff
@@ -193,6 +193,34 @@ static bool IsApxEnabled()
 }
 
 #endif // defined(HOST_X86) || defined(HOST_AMD64)
+
+#if defined(HOST_ARM64)
+static bool IsWindows11OrGreater()
+{
+    // Using RtlGetVersion since GetVersion call can be shimmed on Win8.1+.
+    typedef NTSTATUS (WINAPI *pFuncRtlGetVersion)(PRTL_OSVERSIONINFOW);
+
+    HMODULE hmodNtdll = LoadLibraryA("ntdll.dll");
```

Comment:
```suggestion
    HMODULE hmodNtdll = LoadLibraryW(L"ntdll.dll");
```
The W variants should be preferred.